StyleGuide.md

```diff
@@ -970,6 +975,111 @@ Use the following HTML and Liquid to include an image in a Markdown page:
+
+The syntax is a little hard to read inline, but based on some experimenting it appears that it _must_ be written as one line so as not to introduce line breaks in the resulting text.
+
+Formatted for easier reading, the template code looks like:
+
+```
+{% if page.name == "cost-based-optimizer.md" %}  
+Locality-optimized search  
+{% else %}  
+[Locality-optimized search](cost-based-optimizer.html#locality-optimized-search-in-multi-region-clusters)  
+{% endif %}  
+```
+
+##### Different content depending on Cockroach Cloud or Self-Hosted
+
+Sometimes, you must use an include file on both a Cockroach Cloud and a Self-Hosted page. This requires that the links be resolved differently depending on the page that the include file is being referenced from.
+
+For instructions showing how to use relative links that differ depending on the page type, see [Use relative links to Cloud and Core topics in _include files](https://cockroachlabs.atlassian.net/wiki/x/woBZl).
+
+_Note_: In the previous link, Cloud = Dedicated and Core = Self-Hosted.
+
+<a name="remote-includes"></a>
+
+##### Remote includes
+
+Sometimes you need to include files that are maintained in other places than the `cockroachdb/docs` repo but referenced in our docs. The `remote_include` tag is used for this. We most often use this tag for:
+
+- SQL diagrams, which are maintained in the `cockroachdb/cockroach` repo
+- Code samples, which are maintained in various repos
+
+For SQL diagrams, you remotely include the entire (HTML) file as follows:
+
+```
+{% remote_include https://raw.githubusercontent.com/cockroachdb/generated-diagrams/release-22.1/grammar_svg/show_databases.html %}
+```
+
+For code samples, you usually want to show only part of a larger file to highlight a specific technique, or due to length considerations.
+
+To accomplish this, the `remote_include` tag lets you pass arguments (usually named `START {text}` and `END {text}` by convention) that pull in the text of the remote file between `START {text}` and `END {text}`.
+
+For example, the file `movr-flask-application.md` (which becomes the page [Develop a Global Web Application](https://www.cockroachlabs.com/docs/v22.1/movr-flask-application.html)) has the following `remote_include`:
+
+```
+{% remote_include https://raw.githubusercontent.com/cockroachlabs/movr-flask/v2-doc-includes/dbinit.sql ||-- START database ||-- END database %}
+```
+
+If you browse to the `dbinit.sql` file, you will find the following SQL code block that uses those start and end tags:
+
+```
+-- START database
+CREATE DATABASE movr PRIMARY REGION "gcp-us-east1" REGIONS "gcp-us-east1", "gcp-europe-west1", "gcp-us-west1";
+-- END database
+
```

Comment:
Not sure, but should there be a trailing line at the end of the code block?

Reply:
nope, deleted it - thanks!